### PR TITLE
Add new deploy scripts to circle

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello card](https://trello.com/c/FH1Gbz4J/902-baseline-deployment-for-fb-publisher)

This PR adds the new deploy scripts.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,15 +3,15 @@ jobs:
   test:
     working_directory: ~/circle
     docker:
```
Comment (brenetic) [inline .circleci/config.yml:5]: `&ecr_base_image`

File: .circleci/config.yml [hunk 2]
```diff
@@ -25,34 +25,36 @@ jobs:
   build_and_deploy_to_test:
     working_directory: ~/circle/git/fb-publisher
     docker: &ecr_base_image
```
Comment (brenetic) [inline .circleci/config.yml:27]: *ecr_base_image

File: .circleci/config.yml (hunk 1 above)
Comment (tomas-stefano) [reply .circleci/config.yml:5]: Does it work with more options? The docker block has the environment options too. Let me try.

Comment (brenetic) [reply .circleci/config.yml:5]: ah yes, sorry didn't spot that. no i don't think it does

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**